fivetran_test.go

```diff
@@ -478,6 +479,19 @@ func cleanupWebhooks() {
 	}
 }
 
+func cleanupTeams() {
+	list, err := Client.NewTeamList().Do(context.Background())
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, team := range list.Data.Items {
+		_, err := Client.NewTeamDelete().TeamId(team.Id).Do(context.Background())
+		if err != nil && err.Error() != "status code: 404; expected: 200" {
+			log.Fatal(err)
+		}
+	}
+}
+
```

Comment:
What if amount of teams is more that default limit (>1000)
We should handle the case if next_cursor is not null in response.
Actually the same should be done for all cleanup methods.

Reply:
Simplest way is to make recursive call like :
```
if list.Data.NextCursor != "" {
   cleanupTeams()
}
```

Reply:
Done